Add optional email notifications (#92)
calliope-project/euro-calliope: README.md

To remove cluster results on your local machine, run `snakemake --use-conda clean_cluster_results`.

## Be notified of build successes or fails

>  As the execution of this workflow may take long, you can be notified whenever the execution terminates either successfully or unsuccessfully. Notifications are sent by email. To activate notifications, add the email address of the recipient to the configuration key `email`. For example, you may want to choose running the workflow the following way to receive notifications:

Reviewer: ```suggestion
 As the execution of this workflow may take a while, you can be notified whenever the execution terminates either successfully or unsuccessfully. Notifications are sent by email. To activate notifications, add the email address of the recipient to the configuration key `email`. You can update this in your configuration file, or you may want to choose running the workflow the following way to receive notifications:
```
